### PR TITLE
updated content for CSS-background-opacity

### PR DESCRIPTION
**Pull request by poojajunnarkar11**

### CODE REVIEW

**Comment by scastile** [comment]
add text "Opacity is often used to draw the user’s attention to elements and focus on important parts of a page."

File: src/pages/css/background-opacity/index.md [hunk 1]
```diff
@@ -3,13 +3,12 @@ title: Background Opacity
...
-<!-- Please add any articles you think might be helpful to read before writing the article -->
-
-
+`opacity` is a css property that specifies the opacity or transparency factor of an element.
+```
+img {
+    opacity: 0.5;
+}
+```
+The above code snippet applies 50% opacity to an img element. The value of opacity property can range between 0 to 1. Higher the value of `opactiy`, lower is the transparency of the element.
```
Comment (scastile) [inline src/pages/css/background-opacity/index.md]: Opacity is often used to draw the user’s attention to elements and focus on important parts of a page.

**Comment by Bouncey** [comment]
@poojajunnarkar11 Thanks for the article, however I am closing this PR as it does not conform to our style guide.

Please consult the [Article Style Guide](https://github.com/freeCodeCamp/guides/blob/master/README.md#article-style-guide) before opening another PR for this article.

Many thanks

**Comment by Bouncey** [comment]
@poojajunnarkar11 
It seems that changes have already been accepted for this article, sorry for that. I you feel you have more to add, please open a new PR

Many Thanks

**Pull request closed without merging**